book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
```

Comment:
A method for adding rentals should be present.

Link to an example [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship)

```suggestion

def add_rental(person, date)
# Create new rental
end
```

Reply:
Ok. I will try to add it.

Reply:
Hi, @od-c0d3r. I just created the `add_rental` method. but can I ask you why we need it? I think if we want to define an object of Rental, we have to give the rental object an argument `person`. It means each rental object has a `person` argument. why do we need to have the `add_rental` method for `Person` class?
Let me give you an example: 
```rb
book1 = Book.new("title1", author1")

person1 = Person.new(27, "Amine")

rental1 = Rental.new("01-01-2022", person1, book1)
```
It means there is no `rental1` without `person1`